lib/command/run_detached.rb

```diff
@@ -45,6 +45,7 @@ class RunDetached < Base # rubocop:disable Metrics/ClassLength
     EX
 
     WORKLOAD_SLEEP_CHECK = 2
+    MAX_RETRIES = Float::INFINITY
```

Comment:
I'm not sure we should set infinity at all. There should be a sane default number, possibly configurable with env var.